bin/butane.js

```diff
@@ -1,9 +1,3 @@
 #!/usr/bin/env node
 
-var argv = require('minimist')(process.argv.slice(2));
-var fireRules = require('../dist/');
-
-var input = argv._[0];
-var output = argv._[1];
-
-fireRules.convert(input, output);
+require('require-relative-main')('./cli', __dirname)
```

Comment:
Much appreciated. Happy to spread the modular gospel :)

I have a build tool that I originally wrote for handling UMD builds that I refactored to be a more generic transpilation and tagging utility. Here's the babel bit:

https://github.com/bendrucker/publicist-es5

It changes the main field as part of the build lifecycle. I'll document it and start promoting it more soonish.